client/src/test/resources/logback-test.xml

```diff
@@ -17,7 +17,7 @@
         </encoder>
     </appender>
 
-    <root level="DEBUG">
+    <root level="INFO">
```

Comment:
This sounds like a good change, but out of scope and I'd rather not do it silently as part of this PR.

Reply:
True - I had pushed this change by mistake, and have now reverted it.